components/network/vpnc/Makefile

```diff
@@ -15,21 +15,26 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		vpnc
-COMPONENT_VERSION=	0.5.3
```

Comment:
With REVISION(==1 for the first bump) in place, AFAIK you get something like `...0.5.3.1` which is above any 0.5.3 or 0.5.3.0 but under any 0.5.4.